packages/component-submission/client/pages/SubmissionWizard.js

```diff
@@ -50,15 +50,15 @@ const SubmissionWizard = ({
           handleButtonClick={updateManuscript}
           history={history}
           initialValues={initialValues}
-          nextUrl={`${match.url}/submission`}
+          nextUrl={`${match.url}/details`}
```

Comment:
One nice future improvement is to be able to combine the step names, WizardSteps and TrackedRoute to a common source.Then the render can happen dynamically - rather than an explicit list.